projects/plugins/jetpack/extensions/shared/currencies.js

```diff
@@ -1,5 +1,10 @@
 import { CURRENCIES, getCurrencyDefaults } from '@automattic/format-currency';
-import { trimEnd } from 'lodash';
+
+// Removes all dots (`.`) from the end of a string.
+function removeTrailingDots( string ) {
+	const split = String( string || '' ).split( '.' );
+	return split.slice( 0, split.findLastIndex( i => i !== '' ) + 1 ).join( '.' );
```

Comment:
If we don't think `String.trimEnd` is good enough, we might still make this more straightforward
```suggestion
	return String( string || '' ).replace( /\.+$/, '' );
```

Reply:
🤦 How did I not think of this? This is so much simpler and cleaner, thank you @anomiex! 👍